src/main/webapp/i18n/de/course.json

```diff
@@ -90,6 +90,27 @@
             "upload": "Hochladen",
             "uploading": "Lade hoch...",
             "fileUploadExercises": "Dateiupload-Aufgabe",
+            "search": {
```

Comment:
I don't think `course.search` is an appropriate key here since at least to me it looks initially like a search for courses.

Reply:
All the other translations for the page are in this file, so I thought it was the best place to put my translations here as well. If you have a better idea where I could put them, please let me know.

Reply:
`course.exercise.search` would at least semantically make more sense. I think in the future we might want to rework the translation system to get a more slim system with less redundancy but for know I'd suggest that key.